scripts/ci/validate-schema.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
+# Install kubeval
```

Comment:
When I looked at this a few months back, I ended up settling on kubeconform instead of kubeval - https://github.com/sourcegraph/deploy-sourcegraph/commits/caug/add-kubeval. Can't remember exactly why, maybe because it's the same person who's maintaining that forked schema?

Reply:
Yes you're correct! kubeval died apparently and kubeconform took the spot. Will switch over.